### DISCUSSION TITLE
how to manually perform an update?

### DISCUSSION THREAD

**Discussion by obnoxxx** [ohmybash/oh-my-bash#592] [Q&A] [answered]
How can one initiate an update of installed oh-my-bash?

Sometimes, one does not want to wait for auto-update to get triggered again.

**Comment by akinomyoga** [comment, marked as answer]
> how to manually perform an update?

README has [a section with the exact title "Manual updates"](https://github.com/ohmybash/oh-my-bash/?tab=readme-ov-file#manual-updates). You may run the shell function `upgrade_oh_my_bash`.
